h2o-core/src/main/java/hex/segments/LocalSequentialSegmentModelsBuilder.java

```diff
@@ -54,8 +55,29 @@ void buildModels(SegmentModels segmentModels) {
         SegmentModels.SegmentModelResult result = segmentModels.addResult(segmentIdx, builder, failure);
         Log.info("Finished building a model for segment id=", segmentIdx, ", result: ", result);
         cleanUp(builder);
+        if (result.isSuccessful())
+          stats._succeeded++;
+        else
+          stats._failed++;
       }
     }
+    return stats;
+  }
+
+  static class SegmentModelsStats extends Iced<SegmentModelsStats> {
```

Comment:
Could easily be normal class, not an inner static class. It's not only used in the `LocalSequentialSegmentModelsBuilder.java` builder, but also in `SegmentModelsBuilder` c lass.

Reply:
good point, will do!